src/test/java/tests/ContextMenuTest.java

```diff
@@ -0,0 +1,18 @@
+package tests;
+
+import org.openqa.selenium.Alert;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ContextMenuTest extends BaseTest {
+
+    @Test
+    public void checkСontextMenuMessageTest() {
+        contextMenuPage.openPage();
+        contextMenuPage.clickElementContextMenu();
+        Alert alert =  contextMenuPage.getAlert();
+        String messageAlert = contextMenuPage.getTextForAlert(alert);
+        Assert.assertEquals(messageAlert, "You selected a context menu");
+        contextMenuPage.closeAlert(alert);
```

Comment:
Why you closing alert after you make verifiction (assert) ? I think it's not necessary because anyway you closing your browser after each test. It's extra step.